examples/connectivity/plot_extract_regions_dictlearn_maps.py

```diff
@@ -116,21 +120,22 @@
 
 ################################################################################
 # Showing Default Mode Network (DMN) regions before and after region extraction
-# by manually identifying the index of DMN in ICA decomposed components
+# by manually identifying the index of DMN in Dictionary Learning decomposed
+# components
 from nilearn._utils.compat import izip
 
-# First we plot DMN without region extraction, interested in only index=[3]
-img = image.index_img(components_img, 3)
+# First we plot DMN without region extraction, interested in only index=[4]
+img = image.index_img(components_img, 4)
 coords = plotting.find_xyz_cut_coords(img)
 display = plotting.plot_stat_map(img, cut_coords=((0, -52, 29)),
-                                 colorbar=False, title='ICA map: DMN mode')
+                                 colorbar=False, title='Dictionary map: DMN mode')
 
 # Now, we plot DMN after region extraction to show that connected regions are
 # nicely separated. Each brain extracted region is indicated with separate color
 
 # For this, we take the indices of the all regions extracted related to original
-# ICA map 3.
-regions_indices_of_map3 = np.where(np.array(regions_index) == 3)
+# dictionary map 4.
```

Comment:
'dictionary map' might be confusing because the dictionary refers to the entire set of components, rather than a single of them.